CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ The contributing process in this project relies on the [GitHub flow](https://gui
    - Updating relevant documents if you add, remove, update props, configuration, or themes
```

Comment:
@derberg not sure how to handle the above links? 🤔 apparently the files have been removed.

Reply:
We can remove this `CONTRIBUTING.md` and rather refer to the one located in `https://github.com/asyncapi/.github` repository.

Reply:
@smoya done.